core/server/master/src/main/java/alluxio/master/block/meta/MasterWorkerInfo.java

```diff
@@ -366,17 +346,18 @@ public boolean isRegistered() {
    * @return the map from tier alias to lost storage paths in this worker
    */
   public Map<String, List<String>> getLostStorage() {
-    return new HashMap<>(mLostStorage);
+    return new HashMap<>(mUsage.mLostStorage);
   }
 
   /**
    * @return true if this worker has lost storage, false otherwise
    */
   public boolean hasLostStorage() {
-    return mLostStorage.size() > 0;
+    return mUsage.mLostStorage.size() > 0;
   }
 
   @Override
+  // TODO(jiacheng): Should this be locking?
```

Comment:
@ggezer @apc999 This is potentially harmful but is it too heavy to lock here?

Reply:
ideally with read locking, or string could be inconsistent.

Reply:
The only toString conversion seems to be [here](https://github.com/Alluxio/alluxio/blob/e6b7900126ebb8d236410f35ad69b6ffdb258470/core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java#L914) but i'm not sure if there are more that I don't catch. I'm actually leaving a TODO here because:
1. This PR doesn't change the existing behavior of a unlocked toString()
2. How to guard properly and return a String with as little locking as possible is not on the top of my head. Let's not block this PR with that discussion.

Reply:
Updated the TODO message btw.